test/objectwrap.js

```diff
@@ -182,6 +182,24 @@ const test = (binding) => {
     }
   };
 
+  const testFinalizer = (clazz) => {
+    
+    let finalizerCalled = false;
+    const finalizerCb = function(called){
```

Comment:
Nit: need a space between the ) and the {

Reply:
Got it.